wsagent/che-datasource-api/src/main/java/org/eclipse/che/datasource/api/AvailableDriversService.java

```diff
@@ -0,0 +1,82 @@
+package org.eclipse.che.datasource.api;
+
+import org.eclipse.che.datasource.shared.DriversDTO;
+import org.eclipse.che.datasource.shared.ServicePaths;
+import org.eclipse.che.dto.server.DtoFactory;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import javax.inject.Singleton;
+import javax.ws.rs.GET;
+import javax.ws.rs.Path;
+import javax.ws.rs.Produces;
+import javax.ws.rs.core.MediaType;
+import java.sql.Driver;
+import java.sql.DriverManager;
+import java.util.ArrayList;
+import java.util.Enumeration;
+import java.util.List;
+
+import static org.eclipse.che.dto.server.DtoFactory.newDto;
+
+/**
+ * A service that lists all available JDBC drivers.
+ * @author "Sudaraka Jayathilaka"
+ */
```

Comment:
if this is a copy of https://github.com/codenvy-legacy/plugin-datasource/blob/master/codenvy-ext-datasource-server/src/main/java/org/eclipse/che/ide/ext/datasource/server/AvailableDriversService.java#L32-L36
You should keep the original javadoc and @author. You can add your name as @author if you'd like